packages/node/src/lib/process-response.ts

```diff
@@ -62,12 +58,12 @@ export default function processResponse(
     //
     // This is the same thing that Node.js does internally:
     // https://github.com/nodejs/node/blob/9b8ba2536044ae08a1cd747a3aa52df7d1815e7e/lib/_http_server.js#L318
-    statusText: res.statusMessage || STATUS_CODES[res.statusCode],
+    statusText: (res.statusMessage || STATUS_CODES[res.statusCode]) as string,
     headers: objectToArray(headers, { castToString: true }),
     content: {
       text: JSON.stringify(body),
       size: Number(fixHeader(res.getHeader('content-length') || 0)),
-      mimeType: fixHeader(res.getHeader('content-type')) || 'text/plain',
+      mimeType: fixHeader(res.getHeader('content-type') as string) || 'text/plain',
```

Comment:
same here: `res.getHeader('content-type') || ''` solves the type problem instead of punching holes in the type

Reply:
fixed! https://github.com/readmeio/metrics-sdks/pull/900/commits/8e1b743943a1ebfc03cbdf978d72cebacc8fa226